packages/map/src/pin_marker/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  PIN_MARKER_GREEN,
+  PIN_MARKER_BASE,
+  PIN_MARKER_ORANGE,
+  PIN_MARKER_YELLOW,
+  PIN_MARKER_RED,
+  PIN_MARKER_GRAY,
+  PIN_MARKER_BLUE,
+} from './constants';
+
+import Marker from '../marker';
+
+
+const types = [
+  PIN_MARKER_GREEN,
+  PIN_MARKER_BASE,
+  PIN_MARKER_ORANGE,
+  PIN_MARKER_YELLOW,
+  PIN_MARKER_RED,
+  PIN_MARKER_GRAY,
+  PIN_MARKER_BLUE,
+];
+
+const images = types.reduce((acc, type) => {
+  acc[type] = require(`./images/pin-${type}.svg`);
```

Comment:
use glob import instead?

Reply:
Asked @peterhass to check how we can use it. 
His answer:
> ok it’s not as simple as I thought
> since those components are used within rollup and webpack, I cannot just change the rollup config because webpack wouldn’t work

Will leave it as it is

Reply:
create a ticket please, i think it should be doable to update both configs

Reply:
### Part 1: Not sure if this will work at all

You are using `require` how you would in any of our webpack apps CoreFE, BizFE, etc - you are expecting file-loader / url-loader to return a public url to that asset.

Our webpack apps aren't configured to apply those loaders somewhere in node_modules.
Since we are building a library here we have to approach it differently. 

For this specific case I would use rollups svgo plugin to inline those svgs so that the app using the package doesn't need to concern itself with serving those svgs.

For now I would manually import every single one of those svgs, so that we can be sure that they are processed by rollup. I think rollup wouldn't transform this dynamic require call.

### Part 2: Import glob

Glob import behavior is difficult to control. As soon as you need named exports it gets difficult. I researched a bit and I didn't find a import-glob plugin for webpack which could be configured the same way as `rollup-plugin-glob-import`.

I suggest the following: Let's use `require.context`. It works in webpack out of the box, webpack applies all needed loaders and for rollup we can use `rollup-plugin-require-context`.

Reply:
require.context sounds right to me